internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -577,6 +570,56 @@ func (srv *Server) Serve(Port int) {
 		})
 	})
 
+	r.Route("/ws", func(r chi.Router) {
+		r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+			m.HandleRequest(w, r)
+		})
+	})
+
+	m.HandleMessage(func(s *melody.Session, msg []byte) {
+		params := strings.Split(string(msg), ":")
+		lock.Lock()
+		defer lock.Unlock()
+
+		command := params[0]
+		if command == "" {
+			s.Write([]byte("ERROR"))
+		}
+		bounceRuleID := params[1]
+		if bounceRuleID == "" {
+			s.Write([]byte("ERROR"))
```

Comment:
I think for both of these errors you should exit out immediately, or handle it however else you want. Otherwise the rest of the code in the function gets run. It looks like its okay if `command` is empty since it'll hit the default case on the `switch`. But if `bounceRuleID` is empty and, for example, `command` is `edit` then it will attempt to edit the empty bounce rule ID. Also you need to check `len(params) >= 2` or `len(params) == 2` before moving on, otherwise you'll get segfaults if someone sends you something weird